pkg/util/util.go

```diff
@@ -252,3 +255,11 @@ func Max(x, y int) int {
 	}
 	return x
 }
+
+func CreateLoggerOpts(development bool) k8szap.Opts {
+	return func(o *k8szap.Options) {
+		o.Development = development
+		// use the development encoding so that the timestamp will be displayed in ISO8601 format
+		o.Encoder = zapcore.NewConsoleEncoder(zap.NewDevelopmentEncoderConfig())
```

Comment:
In order to allow configuring the logging via command line params:
```
o.BindFlags(flag.CommandLine)

zapOpts = zap.UseFlagOptions(&opts)

```

Reply:
Oh that comes very handy actually, and we don't need to do any manual configuration at all.
Thanks!